### DISCUSSION TITLE
Better GPU driver for skyline emulator

### DISCUSSION THREAD

**Discussion by Beastboysubb** [skyline-emu/skyline#2077] [Ideas]
You can use opengl GPU driver for skyline it is more stable then vulkan GPU driver and will increase more fps in game  👍👍

You can check the performance in opengl & vulkan in others emulator to compared the performance different 

If you read this message please reply your opinions ❤️❤️

**Comment by JERD001** [comment]
OpenGL support won't ever happen. Vulkan is mandatory for the emulator, same is true for its competitor on mobile.

**Reply by PriyanshuShrestha** [reply]
Best GPU driver for snapdragon 778g

**Reply by PriyanshuShrestha** [reply]
What is best

**Comment by PriyanshuShrestha** [comment]
I need best GPU driver link for snapdragon 778g

**Reply by chostdz** [reply]
Try 615.5 or go to discord server
